bigbluebutton-config/bin/bbb-conf

```diff
@@ -1456,6 +1482,19 @@ if [ $CHECK ]; then
      exit 0
 fi
 
+#
+# Return a list of the running meetings
+#
+if [ $RUNNING_MEETINGS ]; then
+    url=$( get_meetings_url )
+    if [ ! $MACHINE_READABLE ]; then
+        list_of_internalId=$( ruby /usr/local/bigbluebutton/core/lib/xml_helper_conf.rb -u "$url")
```

Comment:
@kepstin, is this the correct folder for the helper file (`/usr/local/bigbluebutton/core/lib/xml_helper_conf.rb`)? Or is there a better place to put it? If that path is ok, I will propagate this change for the build package to  place the `xml_helper_conf.rb` there.

Reply:
I would like to have this script installed into the directory `/usr/libexec/bigbluebutton/`. The `.rb` extension is unnecessary.